<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesMD5Snapshot</class>
 <widget class="QWidget" name="FilesMD5Snapshot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_targetPath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入生成快照的目的文件夹路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>导出对比结果到json中</string>
   </property>
   <property name="statusTip">
    <string>导出对比结果到json中</string>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_buildTip_4">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>目标路径</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_targetPath">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>选择目标路径</string>
   </property>
   <property name="statusTip">
    <string>选择目标路径</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_buildTip_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>对比路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_compared">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入对比文件夹的路径（非必选）</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_compared">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>选择对比路径</string>
   </property>
   <property name="statusTip">
    <string>选择对比路径</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>在表格中搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_withJson">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>是否需要与json文件进行比对，而非某个文件路径</string>
   </property>
   <property name="statusTip">
    <string>是否需要与json文件进行比对，而非某个文件路径</string>
   </property>
   <property name="text">
    <string>与Json对比</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_build">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>导出目标路径中差异化的文件到输出路径</string>
   </property>
   <property name="statusTip">
    <string>导出目标路径中差异化的文件到输出路径</string>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>891</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>显示区</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>871</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>点击表头可以按页签进行排序</string>
    </property>
    <property name="statusTip">
     <string>点击表头可以按页签进行排序</string>
    </property>
    <column>
     <property name="text">
      <string>文件名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>md5值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>对比md5值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否变更</string>
     </property>
    </column>
   </widget>
  </widget>
  <zorder>lineEdit_compared</zorder>
  <zorder>pushButton_export</zorder>
  <zorder>label_buildTip_4</zorder>
  <zorder>toolButton_targetPath</zorder>
  <zorder>label_buildTip_5</zorder>
  <zorder>toolButton_compared</zorder>
  <zorder>lineEdit_targetPath</zorder>
  <zorder>lineEdit_search</zorder>
  <zorder>pushButton_search</zorder>
  <zorder>radioButton_withJson</zorder>
  <zorder>pushButton_build</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
